engine/src/main/java/org/camunda/bpm/engine/impl/db/sql/BatchDbSqlSession.java

```diff
@@ -259,6 +283,19 @@ protected void updateBulk(DbBulkOperation operation) {
     executeUpdate(statement, parameter);
   }
 
+  @Override
+  public int executeUpdate(String updateStatement, Object parameter) {
```

Comment:
To reduce code complexity, I suggest to not override these methods. We will then have the wrapper exception twice in the hierarchy in case an operation fails in these methods when batching is used. I think that is ok.

Reply:
👍